docs/modules/clusters/pages/ucn-security.adoc

```diff
@@ -0,0 +1,14 @@
+= Security
+:description: You can control who can create, deploy, or replace your {ucn} using permissions. You can also provide permissions for the instantiation and registration of user customizations that lookup code in your {ucn}. 
+:page-enterprise: true
+:page-beta: true
+
+{description}
+
+NOTE: Audit events are logged for any {ucn} configuration change.
```

Comment:
Is this true? cc @gareth-johnston @vbekiaris

Reply:
If I haven't missed anything, this is the last thing outstanding comment.

Reply:
Recap from the standup: This functionality does not currently exist, although it's on the backlog (not planned)